test/image/mocks/groups-over-matching-axes.json

```diff
@@ -86,6 +86,9 @@
     }
   ],
   "layout": {
+    "template": {
+      "data": {"violin": [{"bandwidth": 1}]}
```

Comment:
Nice use of `template` 🏆

Reply:
Hmm. I think I would prefer updating the baselines instead of tweaking the JSON mocks.

Reply:
Done in 4698df0

Reply:
Not a big deal, but to me the original baselines are a lot easier to look at and understand what's going on. But of course either way it'll serve the function of catching unintended changes.